src/integrations/flinks.ts

```diff
@@ -23,6 +23,9 @@ export async function openFlinksConnect(
       ? 'toolbox'
       : flinksConfig.metadata.instance;
 
+  const daysOfTransactions =
+    parseInt(flinksConfig.metadata.daysTransactions) || 365;
```

Comment:
instead of hardcoding the default number in the react lib, yous hould instead pass back null to the backend and the backend can set the default number